src/action.ts

```diff
@@ -227,9 +228,9 @@ class Action {
       }
 
       core.error(`Task ${name} with ID ${task.task_id} has failed. Error: ${task.error}`)
```

Comment:
We should switch the message depending on the task status. Saying it `failed` or was `skipped` accordingly, don't you think?